roofit/roostats/src/AsymptoticCalculator.cxx

```diff
@@ -306,7 +306,7 @@ double AsymptoticCalculator::EvaluateNLL(RooAbsPdf & pdf, RooAbsData& data,   co
 
     // need to call constrain for RooSimultaneous until stripDisconnected problem fixed
     auto& config = GetGlobalRooStatsConfig();
-    RooAbsReal* nll = pdf.createNLL(data, RooFit::CloneData(false),RooFit::Constrain(*allParams),RooFit::ConditionalObservables(conditionalObs), RooFit::GlobalObservables(globalObs),
+    RooAbsReal* nll = pdf.createNLL(data, RooFit::CloneData(false),RooFit::ConditionalObservables(conditionalObs), RooFit::GlobalObservables(globalObs),
```

Comment:
Could these be edge cases where `Constrain` still needs to be supported because it was (ab)used before?